api/v2/sdks/client-side-identity.md

```diff
@@ -5,14 +5,36 @@
 Use this UID2 SDK to facilitate the process of establishing client identity using UID2 and retrieving advertising tokens. The following sections describe the high-level [workflow](#workflow-overview) for establishing UID2 identity, provide the SDK [API reference](#api-reference), and explain the [UID2 cookie format](#uid2-cookie-format). 
 
 - For integration steps for content publishers, see [UID2 SDK Integration Guide](../guides/publisher-client-side.md). 
-- For an [example application](https://example-jssdk-integ.uidapi.com/) documentation, see the [UID2 SDK Integration Example](https://github.com/UnifiedID2/uid2-examples/blob/main/publisher/standard/README.md) guide.
-
->NOTE: Within this documentation, the term "identity" refers to a package of UID2 tokens, including the advertising token.
-
+- For an [example application](https://example-jssdk-integ.uidapi.com/), with associated documentation, see the [UID2 SDK Integration Example](https://github.com/UnifiedID2/uid2-examples/blob/main/publisher/standard/README.md) guide.
+
+This guide includes the following information:
+
+- [Terminology](#terminology)
+- [Include the SDK Script](#include-the-sdk-script)
+- [Workflow Overview](#workflow-overview)
+   - [Workflow States and Transitions](#workflow-states-and-transitions)
+   - [Background Token Auto-Refresh](#background-token-auto-refresh)
+ - [API Reference](#api-reference)
+   - [constructor()](#constructor)
+   - [init()](#initopts-object-void)
+   - [getAdvertisingToken()](#getadvertisingtoken-string)
+   - [getAdvertisingTokenAsync()](#getadvertisingtokenasync-promise)
+   - [isLoginRequired()](#isloginrequired-boolean)
+   - [disconnect()](#disconnect-void)
+   - [abort()](#abort-void)
+- [UID2 Cookie Format](#uid2-cookie-format)
+  - [Properties](#properties)
+  - [Contents Structure](#contents-structure)
+
+## Terminology
+
+In this document, the following terms apply:
+- **Identity** refers to a package of values, returned by the [POST /token/generate](../endpoints/post-token-generate.md) or [POST /token/refresh](../endpoints/post-token-refresh.md) endpoint, that includes the UID2 token, the refresh token, and associated values such as timestamps.
+- **Advertising token** refers to the UID2 token.
 
 ## Include the SDK Script
 
-Include the following SDK script on the pages where you want to use UID2 to manage identity or retrieve an advertising token for targeted advertising:
+On the pages where you want to use UID2 to manage identity or to retrieve an advertising token for targeted advertising, include the following SDK script: {**GWH_01/AT: what would "manage identity" consist of here? Can we lose it?**}
```

Comment:
I agree the wording isn't quite right, but I'm not sure what it should say. We really want them to include the script on any page which is shown to users who have a current UID2 advertising token.

Reply:
Thx! I put this:
On every page where you want to use UID2 for targeted advertising, include the following SDK script: